src/Rector/Contrib/PhpParser/IdentifierRector.php

```diff
@@ -68,16 +67,17 @@ public function __construct(
         $this->nodeTypeResolver = $nodeTypeResolver;
     }
 
+    /**
+     * @param PropertyFetch $node
+     */
     public function isCandidate(Node $node): bool
```

Comment:
I see now that it's isCandidate() method. It accepts any node so the annotations here is miss leading

Reply:
Just noticed that as well. Gonna change to an internal `@var`